lib/syskit/shell_interface.rb

```diff
@@ -170,6 +170,45 @@ def disable_log_group(string)
         def disable_logging_of(string)
             disable_log_group(string)
         end
+
+        LoggingGroup = Struct.new(:enabled)
```

Comment:
I would suggest putting the name here as well. It usually helps later, or one ends up passing both the group object and the name at the same time to each functions.